challenge.md

```diff
@@ -5,19 +5,26 @@
 Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
+git checkout response
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
+git log
```

Comment:
`git status` will tell you which files have been altered since the last commit. `git diff` will tell you, line by line, what changes were made to which files. `git log` will list the previous commits. Since you haven't committed your code yet in this scenario, `git log` will not be helpful.